Source/DafnyStandardLibraries/examples/ConcurrentExamples.dfy

```diff
@@ -192,6 +193,16 @@ module ConcurrentExamples {
     expect(b);
   }
 
+  @Test
+  method TestString() {
+    var mmap := new MutableMap(p6);
```

Comment:
I don't understand how we are using `p6`, but the testcase looks fine. I'd say since the Go type is Seq, maybe use more than one character?

Reply:
The `MutableMap` type has a ghost invariant so you can safely assume things about the values in the map. Since we test for multiple `K` and `V` types, we need multiple invariant predicates to match the types.